src/components/editMeasure/measureDetails/MetaDataWrapper.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Button } from "@madie/madie-design-system/dist/react";
+
+const MetaDataWrapper = ({
```

Comment:
I saw this now, we are using it for risk adjustment.

Reply:
I figured I'd build the skeleton for anything else that might end up in the that section since we've been reusing the template so frequently with different content

Reply:
Worst case scenario it just simplifies the render of a single component